fn_falcon_sandbox/fn_falcon_sandbox/util/submit_helper.py

```diff
@@ -0,0 +1,117 @@
+# (c) Copyright IBM Corp. 2010, 2019. All Rights Reserved.
+# -*- coding: utf-8 -*-
+# pragma pylint: disable=unused-argument, no-self-use
+import time
+import requests
+import tempfile
+import os
+from requests_toolbelt import MultipartEncoder
+from resilient_circuits import FunctionError
+from fn_falcon_sandbox.util.constants import (
+    HA_AVAILABLE_ENVS,
+    HA_AVAILABLE_RUNTIME_ACTION_SCRIPTS,
+)
+
+
+def write_temp_file(data, name=None):
+    """This function writes content to a temporary file and 
+        returns file path and list of files.
+    """
+    temp_files = []
+    path = None
+
+    if name:
+        path = "{0}/{1}".format(tempfile.gettempdir(), name)
+    else:
+        tf = tempfile.mkstemp()
+        path = tf[1]
+
+    fo = open(path, "wb")
+    temp_files.append(path)
+    fo.write(data)
+    fo.close()
+    return path, temp_files
+
+
+def remove_temp_files(files):
+    """This function removes all temporary files 
+        created by this function.
+    """
+    for f in files:
+        os.remove(f)
+
+
+def falcon_sandbox_request_header(api_key):
+    """ This function add API Key and 
+        returns request header 
+    """
+    return {
+        "accept": "application/json",
+        "user-agent": "Falcon Sandbox",
+        "api-key": api_key,
+    }
+
+
+def get_submission_queue_size(host_url, api_key):
+    """ This function returns current submission queue size of Falcon Sandbox"""
+
+    headers = falcon_sandbox_request_header(api_key)
+    url = "{}/system/queue-size".format(host_url)
+    response = requests.get(url, headers=headers)
+    response_json = response.json()
+    return response_json["value"]
```

Comment:
* Will `response_json` always have the "value" attribute?
* Should use `response_json.get("value")` so if it does not exist it will return None